locales/cy/what-is-your-role.json

```diff
@@ -2,8 +2,10 @@
     "title": "What is your role? welsh",
     "soleTrader":"I am the sole trader welsh",
     "someoneElse":"I am someone else welsh",
-    "member": "I am a member of welsh ",
-    "memberOfGoverningBody": "I am a member of the governing body of welsh ",
+    "member": "welsh I am a member of ",
+    "memberOfGoverningBody": "welsh I am a member of the governing body of ",
+    "director": "welsh I am a director of ",
+    "generalPartner": "welsh I am a general partner of ",
```

Comment:
Can we follow same pattern of adding 'welsh' at the end of the line please? just to avoid confusion

Reply:
If I add in the end, welsh will appear in between the label like  "I am a director of welsh MORRIS Accounting". I felt its good to be either at the beginning or end of label and not in between.

Reply:
okay, that should be fine then